### PR TITLE
Add log level envar to rasa-pro-services

### PR DESCRIPTION
**Pull request by vcidst**
**Proposed changes**:
- Should be merged after https://github.com/RasaHQ/rasa-pro-services/pull/136

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/deploy/deploy-rasa-pro-services.mdx [hunk 1]
```diff
@@ -119,6 +119,7 @@ environment variables:
 | `RASA_PRO_LICENSE`        | **Required**. The license key for Rasa Pro Services.                                                                  |                    |
 | `KAFKA_BROKER_ADDRESS`    | **Required**. The address of the Kafka broker.                                                                        |                    |
 | `KAFKA_TOPIC`             | **Required**. The topic Rasa Plus publishes events to and Rasa Pro consumes from.                                     | `rasa_core_events` |
+| `LOGGING_LEVEL`           | Set the log level of the application. Valid levels are DEBUG, INFO, WARNING, ERROR, CRITICAL.                         | `INFO`             |
```
Comment (ancalita) [inline docs/docs/deploy/deploy-rasa-pro-services.mdx:122]: We should mention from which version is this available.

Comment (vcidst) [reply docs/docs/deploy/deploy-rasa-pro-services.mdx:122]: Oh then let me do a release for rasa-pro-services

Comment (ancalita) [reply docs/docs/deploy/deploy-rasa-pro-services.mdx:122]: in this case I'd recommend backporting the change to `3.0.x` and making a micro release afterwards.

Comment (vcidst) [reply docs/docs/deploy/deploy-rasa-pro-services.mdx:122]: Should I update on 3.1.x and 3.0.x both?

Comment (ancalita) [reply docs/docs/deploy/deploy-rasa-pro-services.mdx:122]: If both versions are maintained then that would be a great idea 👍🏻

Comment (vcidst) [reply docs/docs/deploy/deploy-rasa-pro-services.mdx]: I've mentioned 3.0.2 on it, when 3.1.x is released I'll also mention that

**Review by ancalita — APPROVED** [review]

**Pull request merged**